### PR TITLE
Merge puzzle and Onramp Widget

### PR DESCRIPTION
**Pull request by kon-rad**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ const nextConfig = {
   images: {
     domains: ['juicebox.money','jbx.mypinata.cloud'],
   },
+  typescript: {
+    ignoreBuildErrors: true,
```
Comment (pi0neerpat) [inline next.config.js:17]: Can we remove this?

File: public/fonts/satoshi/css/satoshi.css [hunk 2]
```diff
@@ -0,0 +1,178 @@
+/**
```
Comment (pi0neerpat) [inline public/fonts/satoshi/css/satoshi.css:1]: Is this font available to use for commercial purposes? Please verify we can use this, and any restrictions/stipulations.

File: components/Fund.tsx [hunk 3]
```diff
@@ -23,6 +24,7 @@ const Fund = () => {
   const { data: session } = useSession();
   const { handleHomePage } = useFormContext();
 
+  console.log("fund here");
```
Comment (pi0neerpat) [inline components/Fund.tsx:27]: remove

File: artifacts/@openzeppelin/contracts/token/ERC1155/ERC1155.sol/ERC1155.dbg.json
Comment (pi0neerpat) [inline artifacts/@openzeppelin/contracts/token/ERC1155/ERC1155.sol/ERC1155.dbg.json:1]: I believe these artifact files should be removed / ignored

File: pages/api/mint.ts
Comment (pi0neerpat) [inline pages/api/mint.ts:1]: Minting should be done via the keyp API

**Review by pi0neerpat — CHANGES_REQUESTED** [review]
It's coming together. Some final comments and changes.

File: artifacts/@openzeppelin/contracts/token/ERC1155/ERC1155.sol/ERC1155.dbg.json
Comment (kon-rad) [reply artifacts/@openzeppelin/contracts/token/ERC1155/ERC1155.sol/ERC1155.dbg.json:1]: Need to import the ABI of the puzzle nft contract to call functions on it

File: components/Fund.tsx (hunk 3 above)
Comment (kon-rad) [reply components/Fund.tsx:27]: yes, good catch. done

File: pages/api/mint.ts
Comment (kon-rad) [reply pages/api/mint.ts:1]: why Keyp API? It needs to be done from this wallet, as it is the 'minter' wallet that is authorized to mint new NFTs to the winner - user that is logged in with Keyp.

File: public/fonts/satoshi/css/satoshi.css (hunk 2 above)
Comment (kon-rad) [reply public/fonts/satoshi/css/satoshi.css:1]: this font was chosen by Sasha in the figma design

File: next.config.js (hunk 1 above)
Comment (kon-rad) [reply next.config.js:17]: yes, I removed it. It was added for quick deployment on vercel, but fixed the TS errors and now will build fine.

File: public/fonts/satoshi/css/satoshi.css (hunk 2 above)
Comment (smombartz) [reply public/fonts/satoshi/css/satoshi.css:1]: > Is this font available to use for commercial purposes? Please verify we can use this, and any restrictions/stipulations.

@pi0neerpat We can use it. https://www.fontshare.com/licenses/itf-ffl

File: pages/api/mint.ts
Comment (pi0neerpat) [reply pages/api/mint.ts:1]: Using the Keyp API to mint was one of the main goals of the project- to demonstrate that you can airdrop to a user with Keyp. However, I'd like to wrap this project up, so no need to worry about it for now.

**Review by pi0neerpat — APPROVED** [review]

**Comment by pi0neerpat** [comment]
Please merge when ready

File: pages/api/mint.ts
Comment (kon-rad) [reply pages/api/mint.ts:1]: Sure, noted, not a  problem, docs also said "Airdrop an NFT to the player after winning." Okay, I'm good with merging it and wrapping up.

**Comment by kon-rad** [comment]
@pi0neerpat I resolved merge conflicts. Someone with write-access can merge now.

**Pull request merged**